packages/editor/src/components/post-title/style.scss

```diff
@@ -49,6 +50,18 @@
 		&:-ms-input-placeholder {
 			color: $dark-opacity-200;
 		}
+
+		&:focus {
+			// Stack borders on mobile.
+			border: $border-width solid transparent;
+			border-left-width: 0;
+			border-right-width: 0;
```

Comment:
This appears to be the cause for a visual regression in the focus style of the title.

Note the lack of right border:

![image](https://user-images.githubusercontent.com/1779930/54823265-b43c8980-4c7d-11e9-97d1-413d8fccaa79.png)

Reply:
👍 PR to fix it https://github.com/WordPress/gutenberg/pull/14572